docs/schema/snitch_cluster.schema.json

```diff
@@ -360,6 +360,12 @@
                                     "description": "Direct memory access (Xdma) custom extension.",
                                     "default": false
                                 },
+                                "xipu": {
```

Comment:
Looks good :) once we split `Xpulpimg` into multiple sub-extensions, I envision this to be an object of multiple booleans.